### DISCUSSION TITLE
wire:click not working

### DISCUSSION THREAD

**Discussion by uxperience-hlaskov** [livewire/livewire#3687] [1. Help] [answered]
Hello, 

I have an issue with wire:click: it does not seem to work, even though all seems OK (styles are there, JS is there). My code snippet is:

&#x3C;div class=&#x22;flex items-center justify-end mt-8&#x22;&#x3E;
    &#x3C;x-jet-button class=&#x22;w-full&#x22; wire:click=&#x22;profile_create&#x22; id=&#x22;submit&#x22;&#x3E;
        {{ __(&#x27;Create profile&#x27;) }}
    &#x3C;/x-jet-button&#x3E;
    &#x3C;div wire:loading wire:target=&#x22;profile_create&#x22;&#x3E;
        Please wait...
    &#x3C;/div&#x3E;
&#x3C;/div&#x3E;

I am using a Jetstream button component. AM i doing something wrong?

Setup is:

Laravel: 8.54
Jetstream: 2.3
Livewire: 2.5

Thanks!

**Comment by denizozturk** [comment, marked as answer]
This is just a component for the jetstream you are using, you need to use this component in a livewire component. Did you create a livewire component? also did you create a function for `profile_create` ?

**Reply by uxperience-hlaskov** [reply]
Hello Deniz, 

Thanks for you input! I figured it out myself - hand't create a livewire component.

Best!
